WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -2218,6 +2248,21 @@ static GSourceFuncs _handlerIntervention =
                 return (Core::ERROR_INCOMPLETE_CONFIG);
             }
 
+#if defined(ENABLE_CLOUD_COOKIE_JAR)
+            if (_config.CryptoProvider.Value().empty() == false) {
+                Exchange::ISimpleCrypto* crypto = service->QueryInterfaceByCallsign<Exchange::ISimpleCrypto>("SimpleCrypto");
```

Comment:
I guess you want to pass the _config.CryptoProvider.Value() as the callsihn here and not the hard coded name ?

Reply:
Do we not want to put this Crypt interface in a plugin notifications activated/deactivated ? What if in the meantime the plugin cryptography is restarted?